src/NCrypt/NCrypt.cs

```diff
@@ -119,13 +119,13 @@ public static partial class NCrypt
         public static extern unsafe SECURITY_STATUS NCryptKeyDerivation(
             SafeKeyHandle hKey,
             NCryptBufferDesc* pParameterList,
-            out byte* pbDerivedKey,
+            byte* pbDerivedKey,
```

Comment:
This is an unrelated bug fix (in its own commit) that I found in the process of developing the code generation since `out byte*` requires special handling (or more accurate: suppression) in code generation.